src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkUtils.java

```diff
@@ -280,6 +280,29 @@ public static String configureBucket(Configuration baseConfiguration,
     return bucket;
   }
 
+  public static void setJsonStringFields(List<BigQueryTableFieldSchema> fields,
```

Comment:
Add javadoc and describe in detail what this method do

Reply:
```java
/**
 * Sets string fields to JSON type if they are present in the provided list of fields.
 * @param fields list of BigQuery table fields.
 * @param jsonStringFields Comma separated list of fields that should be set to JSON type.
 *
 */
 ```